storage/map_files_downloader.cpp

```diff
@@ -161,15 +163,20 @@ MapFilesDownloader::ServersList MapFilesDownloader::LoadServersList()
     request.RunHttpRequest(httpResult);
   }
 
-  std::vector<std::string> urls;
-  downloader::GetServersList(httpResult, urls);
-  CHECK(!urls.empty(), ());
-  return urls;
+  std::optional<MetaConfig> metaConfig;
+  if (httpResult.empty() || !(metaConfig = downloader::ParseMetaConfig(httpResult)))
+  {
+    metaConfig = downloader::ParseMetaConfig(GetPlatform().DefaultUrlsJSON());
```

Comment:
```suggestion
  auto metaConfig = downloader::ParseMetaConfig(httpResult);
  if (!metaConfig)
  {
    metaConfig = downloader::ParseMetaConfig(GetPlatform().DefaultUrlsJSON());
```

Reply:
Yeah, this `httpResult.empty()` is probably excessive now after removal of `abort()` in parser.